curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -346,10 +346,11 @@ int FileCacheManager::Read(uint64_t inodeId, uint64_t offset, uint64_t length,
                     // ret -2 refs s3obj not exist
                     // clear inodecache && get again
                     LOG(INFO) << "inode cache maybe steal, try to get latest";
-                    inodeManager->ClearInodeCache(inodeId);
-                    auto r = inodeManager->GetInode(inodeId, inodeWrapper);
+                    ::curve::common::UniqueLock lgGuard =
+                        inodeWrapper->GetUniqueLock();
```

Comment:
inodeManager->ClearInodeCache(inodeId) It is called in many places（fuse_client.cpp）, does it have the same problem？

Reply:
Other places is used to clear dir‘s inode， as so far， not cause any ploblem， we will fix it later.